app/src/main/java/com/example/poste/models/Post.java

```diff
@@ -47,6 +64,7 @@ private Post(Builder builder) {
         this.description = builder.description;
         this.url = builder.url;
         this.id = builder.id;
+        this.tags = builder.parseTags();
```

Comment:
This can cause a NullPointerException in other locations; make sure we are checking for that when retrieving tags. (Really, the same can happen for the other fields, except they're required on creation so that can't really happen. With tags, they CAN be empty, in which case they won't be set... and then retrieving / iterating over tags will throw that exception)

Reply:
Fixed.
 now you will always get get/set and array lists. the list can be empty, but wont be null